api/src/db-connection.ts

```diff
@@ -61,28 +61,14 @@ export class Connection {
     return await this._knex("logentry").insert(logEntry);
   }
 
-  async getAllPapers(offset: number = 0, size: number = 25): Promise<Paginated<PaperIdWithEntityCounts>> {
-    // We use a local type to capture the fact that counts will come across the wire
-    // as a string.
-    type Row = PaperIdWithEntityCounts & {
-        symbol_count: string;
-        citation_count: string;
-        sentence_count: string;
-        term_count: string;
-        equation_count: string;
-        entity_count: string;
-        total_count: string;
-    };
+  async getAllPapers(offset: number = 0, size: number = 25, entity_type='citations'): Promise<Paginated<PaperIdInfo>> {
```

Comment:
`entity_type` should be typed. You could use [EntityType](https://github.com/allenai/scholarphi/blob/master/api/src/types/api.ts#L79) here.

Reply:
good call, actually highlighted a bug where the type wasn't being applied correctly too